src/components/textbox/textbox.stories.mdx

```diff
@@ -25,14 +24,6 @@ Captures a single line of text.
 import Textbox from "carbon-react/lib/components/textbox";
 ```
 
-## Designer Notes
-
-- Use placeholder text to give the user examples of data formats (e.g. AB123456C for a UK National Insurance number).
-- Use prefixes if your data always begins with a certain sequence (e.g. a UK VAT number usually starts with ‘GB’).
-- If content in a textbox is never editable, think about removing the field border so it appears as static text.
-- You can disable a textbox, but try to avoid this. If you need to, make it clear what the user needs to do in order to activate the textbox.
-- Use wider fields for longer data (e.g. an address line), and narrower fields for shorter data (e.g. a postcode), to give the user a clue about the data expected.
-
```

Comment:
I'm not sure why the designer notes have been removed here, is there some context behind this or was this a mistake?